optimum/commands/export/openvino.py

```diff
@@ -448,10 +452,18 @@ def run(self):
                 from optimum.intel import OVModelForVisualCausalLM
 
                 model_cls = OVModelForVisualCausalLM
-            else:
+            elif "automatic-speech-recognition" in task:
                 from optimum.intel import OVModelForSpeechSeq2Seq
 
                 model_cls = OVModelForSpeechSeq2Seq
+            elif task.startswith("feature-extraction") and library_name in ["transformers", "sentence_transformers"]:
+                from ...intel import OVModelForFeatureExtraction, OVSentenceTransformer
+
+                model_cls = OVModelForFeatureExtraction if library_name == "transformers" else OVSentenceTransformer
```

Comment:
why importing both here ? attempts importing `OVSentenceTransformer` even if `sentence-transformers` is not the lib

Reply:
Do you suggest something like this?

```python
elif task.startswith("feature-extraction") and library_name == "transformers":
    from ...intel import OVModelForFeatureExtraction

    model_cls = OVModelForFeatureExtraction
elif task.startswith("feature-extraction") and library_name == "sentence_transformers":
    from ...intel import OVSentenceTransformer

    model_cls = OVSentenceTransformer
```

Reply:
yes

Reply:
Changed